### DISCUSSION TITLE
How to pay someone back?

### DISCUSSION THREAD

**Discussion by sharadmv** [mmohades/Venmo#56] [Q&A] [answered]
I've been trying to pay people back (i.e. they send me a request and I respond to the request) using the API.

I tried getting the `payment_id` and making a `PUT` to the `/api/payments/{payment_id}` with a JSON payload:
```
{
  "actor": <my user id>,
  "action": "pay",
}
```
and the appropriate auth information. I get a 400 with following error:
```
{"error": {"message": "Invalid parameter : action not available", "code": 104, "links": null, "title": "Error"}}
```
Has anyone been able to do this successfully?

**Comment by mmohades** [comment, marked as answer]
Sorry just seeing this. Did you figure it out?
You may find this helpful:
https://github.com/mmohades/Venmo/blob/master/venmo_api/apis/payment_api.py#L31

**Reply by sharadmv** [reply]
I've been able to get pending payments using the API you suggested, but paying back (or reminding) doesn't seem to work for me.

**Reply by mmohades** [reply]
I believe you are looking for something like this?
```python
from venmo_api import Client, PaymentStatus, AlreadyRemindedPaymentError

api = Client('YOUR TOKEN')
charges = api.payment.get_charge_payments()

for charge in charges:
    if charge.status == PaymentStatus.PENDING:
        print(f"\n\nPending Requested Money:\n{charge}\n\n")
        try:
            api.payment.remind_payment(charge)
        except AlreadyRemindedPaymentError as e:
            print(e)
        continue
    print(charge)
```
